messages/de.json

```
@@ -240,18 +279,24 @@
 	"importFromDelegator": "Konferenz aus Delegator importieren",
+	"importWizardSubtitle": "Konferenz importieren",
 	"imprintAndPrivacy": "Impressum & Datenschutz",
 	"informalCaucus": "Informelle Sitzung",
 	"insertAfterPresentation": "Einfügen nach OP.{index}",
 	"insertAsFirstClause": "Als ersten Absatz einfügen",
 	"insertAtBeginning": "Am Anfang einfügen",
 	"invalidShareCode": "Ungültiger Freigabecode",
 	"italic": "Kursiv",
+	"jumpToStep": "Springen",
 	"language": "Sprache",
 	"launcher": "Launcher",
```

Biome (2.4.14)

[error] The key launcher was already declared.

(lint/suspicious/noDuplicateObjectKeys)